### DISCUSSION TITLE
Bulk Import Products with S3 as Asset Storage

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1526] [Q&A]
My asset storage strategy is set to use S3 bucket to store the files. Would like to understand how the bulk import of product with assets work when the storage is S3 as the document says to copy the asset files to the local asset folder path.

**Comment by michaelbromley** [comment]
You would need to copy the image files to somewhere that is accessible to the import script - ie on the local filesystem or mounted drive. Then, when the import script runs, it will take those image files and move them to S3 as part of the Asset creation process.

**Reply by ramkishore-yerubandi** [reply]
I am using importProducts mutation. In this case, where should i place the images?

**Reply by michaelbromley** [reply]
In this case you'll also need to put them somewhere local to the Vendure server.
